conda-recipe/meta.yaml

```diff
@@ -0,0 +1,78 @@
+{% set setup_py_data = load_setup_py_data(setup_file='../setup.py', from_recipe_dir=True) %}
+
+package:
+  name: ilastik-core
+  version: {{ setup_py_data.version }}
```

Comment:
I wonder whether it's generally a good idea to reference a version from setuptools here...

Conda packages and wheels/sdists are really different things with different dependencies, even though names of those dependencies are often identical. So, maybe we should explicitly specify conda package version here too, like we do with conda requirements? Do we really benefit from using `setuptools_scm`? (I know I advocated for using it, but I'm not so sure anymore, esp. given that we release only on conda).

Reply:
I have to say that I appreciate the convenience that comes with it - we can add automatic builds from master with proper versions and everything in sync without basically any effort. So far `setuptools_scm` has been working nicely for me and I'm happy you recommended it :)